pkg/collector/corechecks/system/wincrashdetect/probe/crashparse.go

```diff
@@ -0,0 +1,135 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+//go:build windows
+// +build windows
+
+package probe
+
+/*
+#cgo LDFLAGS: -l dbgeng -static
+#include "crashdump.h"
+*/
+import "C"
+import (
+	"fmt"
+	"strings"
+	"unsafe"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+type logCallbackContext struct {
+	loglines       []string
+	hasSeenRetAddr bool
+	unfinished     string
+}
+
+const maxscan = int(200)
+
+func min(a, b int) int {
+	if a < b {
+		return a
+	}
+	return b
+}
+
+const (
+	bugcheckCodePrefix     = "Bugcheck code"
+	debugSessionTimePrefix = "Debug session time"
+)
+
+//export logLineCallback
+func logLineCallback(voidctx C.PVOID, str C.PCSTR) {
+	var ctx *logCallbackContext
+	ctx = (*logCallbackContext)(unsafe.Pointer(uintptr(voidctx)))
+	line := C.GoString(str)
+	if !strings.Contains(line, "\n") {
+		ctx.unfinished = ctx.unfinished + line
+		return
+	}
+	if len(ctx.unfinished) != 0 {
+		line = ctx.unfinished + line
+		ctx.unfinished = ""
+	}
+	lines := strings.Split(line, "\n")
+	start := int(0)
+	if !ctx.hasSeenRetAddr {
+		for idx, l := range lines {
+			if strings.HasPrefix(l, bugcheckCodePrefix) {
+				ctx.loglines = append(ctx.loglines, l)
+				return
+			}
+			if strings.HasPrefix(l, debugSessionTimePrefix) {
+				ctx.loglines = append(ctx.loglines, l)
+				return
+			}
+			if strings.HasPrefix(l, "RetAddr") {
```

Comment:
Is there a reason "RetAddr" isn't included in the `const()` section with "Bugcheck code" and "Debug session time"? If no, can it be?